### PR TITLE
Updates for GEOS-Chem 14.3 in CESM

### PR DESCRIPTION
**Pull request by lizziel**
### Name and Institution (Required)

Name: Lizzie Lundgren
Institution: Harvard University

### Describe the update

This PR contains minor updates and fixes to enable building and running GEOS-Chem 14.3 in the CESM model. It must be merged in at the same time as Cloud-J updates in https://github.com/geoschem/Cloud-J/pull/2.

### Expected changes

This is a no diff update.

### Reference(s)

None

### Related Github PR(s)/Issue(s)

- https://github.com/geoschem/Cloud-J/pull/2

### CODE REVIEW

**Comment by lizziel** [comment]
This is ready for review but not ready for merge. I need to make a few updates to Cloud-J standalone for https://github.com/geoschem/Cloud-J/pull/2, and this PR is dependent on that PR.

File: run/CESM/geoschem_config.yml [hunk 1]
```diff
@@ -32,9 +32,33 @@ operations:
...
+      fastjx_input_dir: /see/namelist/file
+      cloudj_input_dir: /see/namelist/file
+    overhead_O3:
+      use_online_O3_from_model: true
+      use_column_O3_from_met: true
+      use_TOMS_SBUV_O3: false
+    photolyze_nitrate_aerosol:
+      activate: true
+      NITs_Jscale_JHNO3: 100.0
+      NIT_Jscale_JHNO2: 100.0
```
Comment (yantosca) [inline run/CESM/geoschem_config.yml:59]: Heads up, the NIT*scale* tags will change in 14.4.0, see PR #1998.

File: run/CESM/HEMCO_Config.rc [hunk 2]
```diff
@@ -1292,287 +1367,295 @@ VerboseOnCores:              root       # Accepted values: root all
 # --- CEDS v2 ---
 #
```
Comment (yantosca) [inline run/CESM/HEMCO_Config.rc:1368]: Heads up: In 14.4.0 there will be further updates for CEDSv2 (diurnal and day-of-week scaling), as well as for 0.1 x 0.1 CEDS emissions.  This section will have to be added for CESM post 14.4.0 release.

File: run/CESM/HEMCO_Config.rc [hunk 3]
```diff
@@ -3168,250 +3536,250 @@ VerboseOnCores:              root       # Accepted values: root all
...
-* GMI_LOSS_ROH        $ROOT/GMI/v2015-02/gmi.clim.ROH.geos5.2x25.nc            loss    2005/1-12/1/0 C xyz s-1   ROH      - 1  1
-* GMI_PROD_ROH        $ROOT/GMI/v2015-02/gmi.clim.ROH.geos5.2x25.nc            prod    2005/1-12/1/0 C xyz v/v/s ROH      - 1  1
-* GMI_LOSS_RP         $ROOT/GMI/v2015-02/gmi.clim.RP.geos5.2x25.nc             loss    2005/1-12/1/0 C xyz s-1   RP       - 1  1
-* GMI_PROD_RP         $ROOT/GMI/v2015-02/gmi.clim.RP.geos5.2x25.nc             prod    2005/1-12/1/0 C xyz v/v/s RP       - 1  1
-* GMI_LOSS_VRO2       $ROOT/GMI/v2015-02/gmi.clim.VRO2.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   VRO2     - 1  1
-* GMI_PROD_VRO2       $ROOT/GMI/v2015-02/gmi.clim.VRO2.geos5.2x25.nc           prod    2005/1-12/1/0 C xyz v/v/s VRO2     - 1  1
-* GMI_LOSS_VRP        $ROOT/GMI/v2015-02/gmi.clim.VRP.geos5.2x25.nc            loss    2005/1-12/1/0 C xyz s-1   VRP      - 1  1
-* GMI_PROD_VRP        $ROOT/GMI/v2015-02/gmi.clim.VRP.geos5.2x25.nc            prod    2005/1-12/1/0 C xyz v/v/s VRP      - 1  1
+* GMI_LOSS_A3O2       $ROOT/v2015-02/gmi.clim.A3O2.geos5.2x25.nc           loss    2005/1-12/1/0 C xyz s-1   A3O2     - 1  1
```
Comment (yantosca) [inline run/CESM/HEMCO_Config.rc:3539]: @lizziel: Should this section be $ROOT/GMI/v2015-02/... ?

**Review by yantosca — CHANGES_REQUESTED** [review]
Hi @lizziel, this mostly looks good.  Can you doublecheck the data paths for the GMI files, I think the paths should be e.g. `$ROOT/GMI/v2015-02/...` etc.

File: run/CESM/geoschem_config.yml (hunk 1 above)
Comment (lizziel) [reply run/CESM/geoschem_config.yml:59]: Good to know. I'll change them when I update CESM to use 14.4.

File: run/CESM/HEMCO_Config.rc (hunk 3 above)
Comment (lizziel) [reply run/CESM/HEMCO_Config.rc]: Yes, good catch! Those are off by default so my tests did not catch that. I just pushed a fix.

File: run/CESM/HEMCO_Config.rc (hunk 2 above)
Comment (lizziel) [reply run/CESM/HEMCO_Config.rc:1368]: Noted. Once we release 14.3.1 and merge it into 14.4.0 dev I can update the config file.

**Comment by lizziel** [comment]
The Cloud-J PR that goes with this PR is now ready for review. https://github.com/geoschem/Cloud-J/pull/2

**Comment by yantosca** [comment]
All GEOS-Chem Classic integration tests passed
```console
==============================================================================
GEOS-Chem Classic: Execution Test Results

GCClassic #1672a99 HEMCO submod update: Merge PR #263 (Restrict volcano clim printout)
GEOS-Chem #9c5803931 Merge PR #2154 (Updates for GEOS-Chem 14.3 in CESM)
HEMCO     #22de25e Merge PR #263 (Restrict volcano clim prints to root thread)

Using 24 OpenMP threads
Number of execution tests: 26

Submitted as SLURM job: 23809135
==============================================================================
 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%  All execution tests passed!  %%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Also, all integration tests were zero-diff w/r/t 14.3.0 except:
- APM (known parallelization issue)
- RRTMG (numerical noise in RRTMG collection file)

**Comment by yantosca** [comment]
After merging https://github.com/geoschem/GCHP/pull/396, all GCHP integration tests passed:
```console
==============================================================================
GCHP: Execution Test Results

GCHP      #6ca8b1f Add new pre-processor definition GCHP_WRAPPER for use in submodules
GEOS-Chem #9c5803931 Merge PR #2154 (Updates for GEOS-Chem 14.3 in CESM)
HEMCO     #22de25e Merge PR #263 (Restrict volcano clim prints to root thread)

Number of execution tests: 6

Submitted as SLURM job: 23864207
==============================================================================
 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%  All execution tests passed!  %%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Also all GCHP integration tests were zero-diff w/r/t 14.3.0.
